docs/excel/excel-add-ins-core-concepts.md

```diff
@@ -32,6 +32,16 @@ Excel.run(function (context) {
 });
 ```
 
+### Run options
+
+**Excel.run** has an overload that takes in a [RunOptions](https://docs.microsoft.com/javascript/api/excel/excel.runoptions) object. This contains a set of properties affecting platform behavior during the function's execution. The following are some of the most commonly used option properties:
```

Comment:
The hyperlink URL for RunOptions should be a relative URL:  `/javascript/api/excel/excel.runoptions`